drop-transfer/src/ws/client/v4.rs

```diff
@@ -156,7 +203,15 @@ impl HandlerLoop<'_> {
         file_id: FileId,
         limit: u64,
     ) -> anyhow::Result<()> {
-        let f = || {
+        let f = async {
```

Comment:
Could you explain this change?

Reply:
I needed to use `.await` inside of lambda and that's the way to go